src/Panel/SearchResultsPanel/SearchResultsPanel.tsx

```diff
@@ -32,6 +41,7 @@ interface SearchResultsPanelProps extends Partial<CollapseProps>{
   actionsCreator?: (item: any) => undefined | ReactNode[];
   /** A renderer function returning a prefix component for each list item */
   listPrefixRenderer?: (item: any) => undefined | JSX.Element;
+  layerStyle?: undefined | LayerStyle;
```

Comment:
Can't you just use `Style` from openlayers?

Reply:
Sure, this works as well and shifts the specific styling to the respective project via openlayers.